style.css

```diff
@@ -50,23 +50,29 @@
 
 .work-section {
   width: 100%;
-  align-items: cente;
+  align-items: center;
```

Comment:
- [ Optional ] Please consider adding this `CSS` rule as the first rule in your stylesheet so that browsers are able to calculate the width, padding, and margins of elements correctly.
```CSS
  * {
     box-sizing: border-box;
  }
```